src/client/src/actions/wizardInfoActions/typeKeys.ts

```diff
@@ -5,5 +5,6 @@ export enum WIZARD_INFO_TYPEKEYS {
   UPDATE_TEMPLATE_GENERATION_STATUS_MESSAGE = "WTS/postgen/UPDATE_TEMPLATE_GENERATION_STATUS_MESSAGE",
   UPDATE_TEMPLATE_GENERATION_STATUS = "WTS/postgen/UPDATE_TEMPLATE_GENERATION_STATUS",
   RESET_WIZARD = "RESET_WIZARD",
-  UPDATE_DEPENDENCY_INFO = "WTS/dependency/UPDATE_DEPENDENCY_INFO"
+  UPDATE_DEPENDENCY_INFO = "WTS/dependency/UPDATE_DEPENDENCY_INFO",
+  UPDATE_CREATE_TEMPLATE_BUTTON = "UPDATE_CREATE_TEMPLATE_BUTTON"
```

Comment:
isn't it CREATE_PROJECT_BUTTON Now? Many things have to follow suit as well (IUpdateCreateTemplateButton and others)